database/imgDataGenerator.js

```diff
@@ -3,45 +3,76 @@ const faker = require('faker');
 
 const randomNumber = (min = 0, max = 9) => Math.floor(Math.random() * (max - min + 1)) + min;
 
-const images = (num) => `https://s3-us-west-1.amazonaws.com/sdcfoodphotos/Food+Images/PIcture${num}.jpg`;
-// 2018-01-25
+function randomDate(start, end) {
+  var d = new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime())),
+    month = '' + (d.getMonth() + 1),
+    day = '' + d.getDate(),
+    year = d.getFullYear();
+
+  if (month.length < 2) month = '0' + month;
+  if (day.length < 2) day = '0' + day;
 
-const date = `2018-${randomNumber(0,12)}-${randomNumber(0,31)}`;
+  return [year, month, day].join('-');
+}
+
+const images = (num) => `https://s3-us-west-1.amazonaws.com/sdcfoodphotos/Food+Images/PIcture${num}.jpg`;
 
 const createImgs = () => {
-  const headers = 'index,ownerId,businessId,title,description,date,url\n';
+  const headers = 'index,userId,businessId,title,description,url,thumbnail,date\n';
   let count = 0;
-  while (count < 10000000) {
+  let picsPerBiz = 1;
+  let numReset = 0;
+
+  const assignImages = (currentCount) => {
+    if (currentCount <= 10000000) {
+      return currentCount;
+    } else {
+      if (numReset === 10000000 && picsPerBiz < 3) {
+        numReset = 0;
+        picsPerBiz++;
+      }
+      if (numReset < 10000000 && picsPerBiz < 3) {
+        numReset++;
+        return numReset;
+      }
+      if (picsPerBiz === 3) {
+        return randomNumber(1, 10000000);
+      }
+    }
+  }
+
+  while (count < 50000000) {
     let storage = ``;
-    let chunkNum = 0;
+    let chunkNum = 0;  
     if (count === 0) {
       storage += headers;
+      count++
     }
-    while (chunkNum < 100000) {
-      let url = images(randomNumber(0, 10));
+    while (chunkNum < 200000) {
+      let url = images(randomNumber(1, 1000));
+      let date = randomDate(new Date(2018, 0), new Date(2018, 12));
+      let currentId = assignImages(count);
       //idx
       storage += `${count},`;
-      //owner id
-      storage += `${randomNumber(0, 10000000)},`;
+      //user id
+      storage += `${currentId},`;
```

Comment:
Since you're doing a template literal you can do this as a one liner; it looks neater that was to me, but if this way looks neater to you by all means keep it